pkg/fileutils/paths_test.go

```diff
@@ -0,0 +1,59 @@
+package fileutils_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/projectriff/riff/pkg/fileutils"
+)
+
+var _ = Describe("StartsWithHomeDirAsTilde", func() {
```

Comment:
How about adding a test for `returns false when starting with ~\ on macOS`?

Reply:
👍